.vscode/launch.json

```diff
@@ -17,7 +17,6 @@
       "name": "Run All Tests",
       "type": "node",
       "request": "launch",
-      "protocol": "inspector",
```

Comment:
interesting! what did this change do?

Reply:
Oh!  I removed this because the linter was beefing "property not allowed":

<img width="380" alt="2022-11-15_14-38-31" src="https://user-images.githubusercontent.com/46458081/202040149-3072466a-b52b-49d8-b71e-0cdfb454cf87.png">

But I'm glad you commented, because it looks like it is a known issue: https://github.com/microsoft/vscode/issues/32202

Since it was there on the original config for Run All Tests, I added it back in, and put it in the config for Run Current Test also.  The linter still beefs, but its only a warning.